os/include/stdlib.h

```diff
@@ -119,7 +119,10 @@ struct mallinfo {
 	int uordblks;				/* This is the total size of memory occupied by
 								 * chunks handed out by malloc. */
 	int fordblks;				/* This is the total size of memory occupied
-								 * by free (not in use) chunks.*/
+								 * by free (not in use) chunks.
+								 *   NOTE : The available size to allocate is smaller than
+ 								 *    the return of this API, because of the size of alloc node.
+ 								 *   available size = free size - SIZEOF_MM_ALLOCNODE*/
```

Comment:
Because it already has freenode, it should be `free size - (SIZEOF_MM_ALLOCNODE - SIZEOF_FREENODE)`. Could you check this?

Reply:
> Because it already has freenode, it should be `free size - (SIZEOF_MM_ALLOCNODE - SIZEOF_FREENODE)`. Could you check this?

The first node in the heap is free node, it should be `free_size - (SIZEOF_MM_ALLOCNODE + SIZEOF_MM_FREENODE)`.
I will change like this.
Thanks.

Reply:
For largest free node, the available size can be deducted to `SIZEOF_MM_ALLOCNODE` only.